From a92d1b947f1d88390bd09a01dc5ad1a1bcfe7be8 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 1 Sep 2016 09:56:13 +0800
Subject: [PATCH] ARM64: dts: rk3399-android: add ddr devfreq nodes for rkfb

Devfreq will register notify to rkfb, to handle vop during ddr
changing frequency.

Change-Id: I22365597054b2155ef1b9754d6ecac243520b3ee
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index a88b8945b908..faa71bda3795 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -320,6 +320,7 @@
 		rockchip,pwr18 = <0>;
 		rockchip,iommu-enabled = <1>;
 		power-domains = <&power RK3399_PD_VOPB>;
+		devfreq = <&dmc>;
 	};
 
 	vopb_mmu_rk_fb: vopb-mmu {
@@ -345,6 +346,7 @@
 		rockchip,pwr18 = <0>;
 		rockchip,iommu-enabled = <1>;
 		power-domains = <&power RK3399_PD_VOPL>;
+		devfreq = <&dmc>;
 	};
 
 	vopl_mmu_rk_fb: vopl-mmu {
-- 
2.35.3

